Automattic/newspack-theme
# feat: add tiktok, telegram icons

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds icons for TikTok and Telegram to the Social Links menu. 

Closes #1510, #722.

### How to test the changes in this Pull Request:

1. Add links to Telegram (t.me) and TikTok (tiktok.com) to the social menu.
2. View on the front end and see that they're both using the placeholder 'link' icons.
3. Apply the PR.
4. Refresh the site; confirm that they're now both using the correct icons (paper plane and quarter note, respectively!):

![image](https://user-images.githubusercontent.com/177561/133295086-0071d30f-82ba-4c22-be8f-7211ae80f929.png)

### Other information:
